dotCMS/src/main/webapp/html/portlet/ext/common/edit_versions_inc.jsp

```diff
@@ -65,92 +65,85 @@
 <div class="contentIdentifier">
 	<%= LanguageUtil.get(pageContext, "Identifier") %> : <%=(ident!=null?ident.getId():"") %>
 </div>
-<table class="listingTable">
-	<tr>
-		<th width="5%" nowrap><%= LanguageUtil.get(pageContext, "Status") %></th>
-		<th width="10%" nowrap><%= LanguageUtil.get(pageContext, "Action") %></th>
-		<% if(isContentlet){ %>
-			<th style="min-width:120px;" nowrap>&nbsp;</th>
-		<% } %>
-		<th width="45%"><%= LanguageUtil.get(pageContext, "Title") %></th>
-		<th width="20%"><%= LanguageUtil.get(pageContext, "Author") %></th>
-		<th width="20%" style="text-align:center;"><%= LanguageUtil.get(pageContext, "Modified-Date") %></th>
-		<th width="20%" style="text-align:center;"><%= LanguageUtil.get(pageContext, "Inode") %></th>
-	</tr>
-<%
-	Iterator<Versionable> versionsIt = versions.iterator();
-	int kmod = 0;
-	boolean isAlreadyLocked = false;
-	while (versionsIt.hasNext()) {
-		Versionable ver = versionsIt.next();
-		boolean working = ver.isWorking();
-		boolean live = ver.isLive();
-		Language langV = APILocator.getLanguageAPI().getDefaultLanguage();
-		if(isContentlet){
-			langV = APILocator.getLanguageAPI().getLanguage(((Contentlet)ver).getLanguageId());
-		}
-		String vinode = ver.getInode();
-		String title = ver.getTitle();
-		String modUser = ver.getModUser();
-		Date modDate = ver.getModDate();
-		String statusIcon = com.dotmarketing.util.UtilHTML.getVersionStatusIcons(ver);
-		String str_style = "";
-		if ((kmod % 2) == 0) {
-			str_style = "class='alternate_1'";
-		}
-		else{
-			str_style = "class='alternate_2'";
-		}
-		kmod++;
+<div style="overflow: hidden">

```

Comment:
Not sute why do we need this but If we need it, it's better to create a class for this edit_versions portlet and do whatever it needs to be done.